Marlin/pins_RAMPS.h

```diff
@@ -350,26 +379,43 @@
       #define DOGLCD_A0         27
       #define LCD_BACKLIGHT_PIN 33
     #elif ENABLED(MINIPANEL)
-      #define BEEPER_PIN        42
-      // Pins for DOGM SPI LCD Support
-      #define DOGLCD_A0         44
-      #define DOGLCD_CS         66
-      #define LCD_BACKLIGHT_PIN 65 // backlight LED on A11/D65
-      #define SDSS              53
+      #if ENABLED(MKS_MINI_12864)
```

Comment:
If the pins are completely different, why nest here ?

Also - indention - it doesn't look like you un-indent this nest.  Close your nests in the endif comment or it's too hard to read:
// MINIPANEL

Reply:
ok